test/App.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { shallow, mount, render } from 'enzyme';
+import App from '../client/src/components/App.jsx';
+
+test('expects App to have state', () => {
+  const wrapper = shallow(<App />);
+  expect(wrapper.state().reviews).toBeDefined();
```

Comment:
what's the difference between `toBeDefined()` and `not.toBeUndefined()`, are those the same thing? (if yes may be worth just using toBeDefined() across all 3 to keep things consistent & avoid double negatives)